### PR TITLE
[FSDE-1174] Update the Chi-Vue boilerplate with new version of CHI-VUE

### PR DESCRIPTION
**Pull request by aryaroudi**
hey @janibolkvadze @jllr please review this

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c8974305a2f8425d29cf6863c13025878acdd68d-chi-pr.enterprise.ctl.io

File: src/boilerplates/vue/chi-vue-boilerplate/vue.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        poll: process.env.WEBPACK_POLL,
+      },
+      overlay: {
+        warnings: true,
+        errors: true,
+      },
+      public: 'localhost:8080',
+      port: 8080,
+      https: true,
+    },
```
Comment (janibolkvadze) [inline src/boilerplates/vue/chi-vue-boilerplate/vue.config.js:36]:
```suggestion
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://82fb7c3f0129241e706ff2010548635fbeea5c75-chi-pr.enterprise.ctl.io

**Review by janibolkvadze — APPROVED** [review]

File: src/boilerplates/vue/chi-vue-boilerplate/package.json [hunk 2]
```diff
@@ -14,13 +14,15 @@
     "vue-property-decorator": "^8.4.2"
   },
   "devDependencies": {
+    "@centurylink/chi-vue": "1.0.0",
```
Comment (jllr) [inline src/boilerplates/vue/chi-vue-boilerplate/package.json:17]:
```suggestion
    "@centurylink/chi-vue": "1.0.1",
```

File: src/boilerplates/vue/chi-vue-boilerplate/src/components/Header.vue [hunk 3]
```diff
@@ -70,14 +70,16 @@
                 <div class="chi-badge -primary -xs">2</div>
               </div>
             </button>
```
Comment (jllr) [inline src/boilerplates/vue/chi-vue-boilerplate/src/components/Header.vue:72]: @aryaroudi Tooltip is missing in notifications btn. Can you add it?

File: src/boilerplates/vue/chi-vue-boilerplate/vue.config.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+    configureWebpack: config => {
```
Comment (jllr) [inline src/boilerplates/vue/chi-vue-boilerplate/vue.config.js:4]: @aryaroudi indentation looks a little off, please take a look.

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/7/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/8/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/9/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/10/. :x:

File: src/boilerplates/vue/chi-vue-boilerplate/vue.config.js (hunk 4 above)
Comment (aryaroudi) [reply src/boilerplates/vue/chi-vue-boilerplate/vue.config.js:4]: hey @jllr I don't think so, looks good on my side 
image

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/11/. :x:

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/12/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-109/13/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://12e7f6b6ec72244c1d2c6bbb32cc6994d0bffc4a-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Pull request merged**